kasa/smartdevice.py

```diff
@@ -27,6 +27,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# Certain module queries will crash devices; this list skips those queries
+MODULE_EXCLUSIONS = {
```

Comment:
```suggestion
MODEL_MODULE_SKIPLIST = {
```
maybe something like this to make the intent clearer?